### ISSUE TITLE
Strange bug with addSeries and Highmaps

### ISSUE THREAD

**Issue by mathielen** [highslide-software/highcharts.com#3940]
When using addSeries with Highmaps and the default "custom/world-continents"-map you cannot add values for any more than 5 continents!
If you add data for 5 or less continents OR do not use addSeries but add the data directly to the config-json - it works.... 

Please - Any idea on that one?
http://jsfiddle.net/mathielen/rLrqvLc1/3/

Thank you!

**Comment by mathielen** [comment]
Its getting stranger:
If I add a "fake"-continent to the .geo.json file like this:

```json
...
    {
      "type": "Feature",
      "id": "XX",
      "properties": {
        "hc-group": "admin0",
        "hc-key": "xx",
        "hc-a2": "XX",
        "abbrev-a3": "XXX",
        "iso-a2": "XX",
        "name": "X X"
      },
      "geometry": {
        "type": "MultiPolygon",
        "coordinates": []
      }
    }
```

It works with 5+ continent values!

**Comment by pawelfus** [comment]
Fixed since 1.1.5: http://jsfiddle.net/rLrqvLc1/4/
